tools/cldr-apps/js/src/views/DashboardPanel.vue

```diff
@@ -106,6 +110,34 @@ export default {
     this.fetchData();
   },
   methods: {
+    scrollto: function scrollto(event) {
+      const whence = event.target.getAttribute("notification");
+      if (this.data && this.data.notifications) {
+        for (let n of this.data.notifications) {
+          if (n.notification == whence) {
+            n.hidden = false;
+            const selector = `#notification-${whence}`;
+            try {
+              const scrollFix = event.target.offsetParent.offsetTop;
+              document.querySelector(selector).scrollIntoView(true);
+              // Now, scroll BACK because:
```

Comment:
possible alternative: `scrollIntoView({ block: "center" })`

Reply:
Like to get this in and iterate.

Reply:
Updating the browser hash would break as you might imagine.